Dockerfile.cross

```
@@ -1,12 +1,12 @@
-FROM __BASEIMAGE_ARCH__/alpine:3.15
+FROM __BASEIMAGE_ARCH__/alpine:3.22
 MAINTAINER Jonathan Weisberg <jo.weisberg@gmail.com>
 
 # Git source https://github.com/phpvirtualbox/phpvirtualbox.git
 __CROSS_COPY qemu-__QEMU_ARCH__-static /usr/bin
 RUN apk update && apk upgrade
-RUN apk --no-cache --update add bash tzdata nginx php8-fpm php8-cli php8-common php8-json php8-soap php8-simplexml php8-session \
+RUN apk --no-cache --update add bash tzdata nginx php84-fpm php84-cli php84-common php84-json php84-soap php84-simplexml php84-session \
     && apk --no-cache --update add --virtual build-dependencies wget unzip \
-    && wget --no-check-certificate https://github.com/phpvirtualbox/phpvirtualbox/archive/master.zip -O phpvirtualbox.zip \
+    && wget --no-check-certificate https://github.com/phpvirtualbox/phpvirtualbox/archive/7.2-dev.zip -O phpvirtualbox.zip \
```

It's now released at https://github.com/phpvirtualbox/phpvirtualbox/releases/7.2-2. 



```suggestion
    && wget --no-check-certificate https://github.com/phpvirtualbox/phpvirtualbox/archive/7.2-2.zip -O phpvirtualbox.zip \
```